src/ice/candidate.rs

```diff
@@ -205,7 +205,11 @@ impl Candidate {
     ///
     /// Relayed candidates are server sockets relaying traffic to a local socket.
     /// Allocate a TURN addr to use as a local candidate.
-    pub fn relayed(addr: SocketAddr, proto: impl TryInto<Protocol>) -> Result<Self, IceError> {
+    pub fn relayed(
+        addr: SocketAddr,
+        base: SocketAddr,
```

Comment:
I think this function should document, what `base` and `addr` are.

Reply:
check